Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -81,6 +81,11 @@ open class ChatChannelListVC: _ViewController,
     private lazy var listChangeUpdater: ListChangeUpdater = CollectionViewListChangeUpdater(
         collectionView: collectionView
     )
+
+    /// Currently there are some performance problems in the Channel List which
+    /// is impacting the message list performance as well, so we skip channel list
+    /// updates when the channel list is not visible in the window.
+    private var skippedRendering = false
```

Comment:
That's a bit sketchy... Is this meant to stay forever or just temporarily?

Reply:
For now, it is temporary, yes, this was making the message list totally unusable with 100+ messages, especially if the stress test script was running at the same time. If we fix the channel list later, we probably can remove this all together.